### PR TITLE
Include videos for testing TwitterScraper

### LINKED ISSUE #1213: Introduce test cases for Twitter videos
### Short Description
A test for extracting video URL from tweet would be helpful in ensuring consistency of `TwitterScraper`.

### PR DESCRIPTION
**Pull request by singhpratyush**
### Short description

Fixed #1213.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: src/org/loklak/objects/MessageEntry.java [hunk 1]
```diff
@@ -691,6 +691,14 @@ public static String html2utf8Custom(String str) {
         return clean.toString().trim().replaceAll(" +", " ");
     }
 
+    public Collection<String> getVdeos() {
```
Comment (singhpratyush) [inline src/org/loklak/objects/MessageEntry.java]: That was a typo. Fixed it.

File: test/org/loklak/harvester/TwitterScraperTest.java [hunk 2]
```diff
@@ -147,6 +143,7 @@ public void testSimpleSearch() {
         assertEquals(tweet.getText(), tweet_check.get("text"));
         assertEquals(tweet.getPlaceId(), tweet_check.get("place_name"));
         assertEquals(tweet.getPlaceName(), tweet_check.get("place_id"));
+        assertEquals(tweet.getVdeos().toString(), tweet_check.get("videoString"));
```
Comment (kavithaenair) [inline test/org/loklak/harvester/TwitterScraperTest.java]: Here too. 😄

Comment (singhpratyush) [inline test/org/loklak/harvester/TwitterScraperTest.java]: I had changed it locally. Don't know why it isn't showing up here.

Comment (singhpratyush) [inline test/org/loklak/harvester/TwitterScraperTest.java]: Made the changes.

**Comment by singhpratyush** [comment]
@hemantjadon: I have already added the `.editorconfig` file in #1209. All the files that I open in my editor are changed according to the `.editorconfig`.

How to apply the linting changes to all files at once?

**Comment by hemantjadon** [comment]
@singhpratyush https://www.npmjs.com/package/editorconfig-tools You might find this useful for once fixing all the errors. I guess this is not related to this PR. But still, linting and indentation should be fixed. This might be the way to go for it.

**Comment by hemantjadon** [comment]
Also some similar package may be hooked into travis to check that the editorconfig is satisfied by all  the PR's.

**Pull request merged**